run_robot_test.sh

```diff
@@ -41,6 +69,6 @@ ${VENV_ROOT}/bin/pip install -r requirements.txt
 tmp_dir=$(mktemp -d -t ods-ci-$(date +%Y-%m-%d-%H-%M)-XXXXXXXXXX)
 #TODO: Configure the "tmp_dir" creation so that we can have a "latest" link
 mkdir $tmp_dir
-./venv/bin/robot -d $tmp_dir -x xunit_test_result.xml -r test_report.html --variablefile test-variables.yml tests/Tests/test.robot
+./venv/bin/robot -d $tmp_dir -x xunit_test_result.xml -r test_report.html ${TEST_VARIABLES} --variablefile ${TEST_VARIABLES_FILE} ${TEST_CASE_FILE}
```

Comment:
can we also keep the original --variablefile option(via if/else)? This helps running the test locally or else we have to set it in cli everytime to run.

Reply:
The previous functionality is still preserved based on the default values for each of the new bash variables. If the is no cli override then the script will default to `--variablefile test-variables.yml tests/Tests/test.robot` once the bash variables are processed.